src/main/java/org/ilgcc/app/utils/enums/FileNameUtility.java

```
@@ -53,11 +55,18 @@ public static String formatApplicantNameForFileName(String fullNameLastToFirst)
         return String.join("-", nameParts);
     }
 
-    public static String getSharePointFilePath(Submission submission, String processingOrg) {
-        return String.format("/%s/%s/%s",
-                processingOrg,
-                getDashFormattedSubmittedAtDate(submission),
-                formatApplicantNameForFileName(getApplicantNameLastToFirst(submission) + "-" +
-                        getDashFormattedSubmittedAtDateWithTime(submission)));
+    public static String getSharePointFilePath(Submission submission, Environment environment) {
+        String ccrrslug = CCRR.getCCRRSlugByOrganizationId(submission.getInputData().getOrDefault("organizationId", "").toString(), environment);
+        if(ccrrslug.isEmpty()){
+            log.error(String.format("Cannot find CCRR for submission: %s", submission.getId()));
```

How to handle a missing slug

---

This could occur if we have active submissions that were created before we set organization code. One option is to do a data migration to load the organization id.

Another option is to set a default slug and log an issue if this ever happens.

I am impartial to the first option because it will mean our data is clean moving forward. And, I think if there is no slug, we should raise an exception